engine/src/test/java/com/redhat/cloud/notifications/templates/TestRbacTemplate.java

```diff
@@ -115,6 +116,9 @@ private void testTitle(String eventType, String result) {
             case REQUEST_ACCESS:
                 assertEquals("Instant notification - Request access - User Access - Console", result);
                 break;
+            case RH_TAM_ACCESS_REQUESTED:
+                assertEquals("Instant notification - TAM access requested - User Access - Console", result);
```

Comment:
```suggestion
                assertEquals("Instant notification - New TAM access request created - User Access - Console", result);
```
It have to be the exact same content as `engine/src/main/resources/templates/Rbac/tamAccessRequestEmailTitleV2.txt`

Reply:
Oh, thanks for catching that! I looked through the log but did not find where it was failing. Updated